src/app/markdown/markdown.component.ts

```diff
@@ -42,7 +42,7 @@ export class MarkdownComponent implements OnChanges, OnInit, AfterViewChecked {
   boxHeight: number;
   enableShowMore: boolean = false;
   // these need to be public for the tests accessing them.
-  renderedText = '';
+  renderedText: any = '';
```

Comment:
Can we use `SafeHtml` instead of `any`?

Reply:
Same as above, coulbe be string or SafeValue. In fact, it will in most cases be just string. Only when someone wants to use extended HTML, he needs to wrap it in SafeValue. This way we give the user the option to either use it with sanitized HTML or with non-sanitized HTML.